### ISSUE TITLE
construction=* tag is ambigous

### ISSUE THREAD

**Issue by d1g** [openstreetmap/iD#2830]
Minor issue in presets 1.8.0.
http://taginfo.openstreetmap.org/tags/railway=construction#combinations
http://taginfo.openstreetmap.org/tags/landuse=construction#combinations
http://taginfo.openstreetmap.org/tags/highway=construction#combinations

Currently iD will display highway subtags with landuse preset. No effect with railway preset.

Probably we want separate template for laduses. AFAIK construction wasn't clearly defined/documented.
Probably we want separate template for railways, since they were explained at wiki and there a few of them (~5932)

**Comment by bhousel** [comment]
Can you post a screenshot of the issue?

**Comment by d1g** [comment]
Probably it is because **presets.fields.construction.label** was previously only used with highway=* and I decided to translate it more verbosely, but recently this text string was reused with landuse=* preset. And here user see confusing text "type of the constructed road" instead of just "type"

note landuse=* preset but con:
http://imgur.com/7g8rkUB


If you search in presets with **TEXT: Type**, you will get 25-35 strings with single word "Type" but it isn't clear what translators should do about it? Should we adapt strings more naturally or should we just copy&paste translations 30 times and why?

My guess was that "Type" strings are for more natural/verbose text labels. What should we do now?

**Comment by jfirebaugh** [comment]
See "Why are there so many duplicate "Type" translations?" in the [Translation section of CONTRIBUTING.md](https://github.com/openstreetmap/iD/blob/master/CONTRIBUTING.md#translating).

**Comment by d1g** [comment]
True, I missed this document. Then "Тип" will be enough for most or all cases.

But it is still true that suggested values in construction=* select field are confusing
https://github.com/openstreetmap/iD/blob/master/data/presets/presets.json#L4896

iD will suggest mixed (light_rail, house, motorway_link) values 
http://imgur.com/YyZywCI


Since [taginfo isn't so helpful](http://taginfo.openstreetmap.org/keys/construction#combinations) let's dig database. Stats for "construction=light_rail global":
    "nodes": 16,
    "ways": 557,
    "relations": 1,

Stats for "**landuse=construction** and construction=light_rail global" http://overpass-turbo.eu/s/cGN
    **"total": 24,**
    "nodes": 0,
    "ways": 24,
    "relations": 0,

Stats for "**railway=construction** and construction=light_rail global" http://overpass-turbo.eu/s/cGO
    "total": 459,
    "nodes": 0,
    **"ways": 458,**
    "relations": 1,

In other words, construction=light_rail should be used mostly/only with railway=* presets. Thats why I called issue "construction=* ambiguous". Is it more clear now?

**Comment by bhousel** [comment]
> In other words, construction=light_rail should be used mostly/only with railway=* presets. Thats why I called issue "construction=* ambiguous". Is it more clear now?

I think I understand, but I can't think of a better way to filter the suggestion list in the `construction=*` field.  That field can contain anything, and we populate the suggestion list from the most popular values in taginfo.

You might say, "But I drew an area, so I only want to see values that appear on areas".  And taginfo does let us filter by type - but OpenStreetMap doesn't have a true area type - so we would only be able to show you all of the `way` and `area` values, so this isn't a very good filter.

You might say, "If I already have `landuse=construction`, taginfo should be able to fetch values for `construction=*` that appear in combination with `landuse`."  And it can, but by the time you filter combinations of 2 keys, you are left with values that are not very popular (as you found above - only a few dozen to a few hundred uses).

So after thinking about this a lot, I think it's best to just leave the `construction=*` field as is..
